src/get-started/write-and-run-unit-tests.md

```diff
@@ -127,15 +127,17 @@ When you write complex unit tests, you may take a look at what test kits we have
 
 ### Running all tests
 
-You can always run all tests by executing the `gotest` target in Makefile:
+We have two ways to run all tests. The first one is to use `go test` directly:
 
 ```
-make gotest
+make ut
 ```
 
-This is almost equivalent to `go test ./...` but it enables and disables fail points before and after running tests.
+The second one is to use bazel to run tests. The bazel will cache the test results and it is faster than the first one. But it need more storage space to save the cache. it is recommended to use the first one in the CI.
```

Comment:
```suggestion
The second one is to use bazel to run tests. Bazel will cache the test results and it is faster than the first one. But it needs more storage space to save the cache.
```

Also, I don't understand what " it is recommended to use the first one in the CI." means: you're talking about the second approach but now said the first approach is recommended in CI? Or what do you want to mean?